### PR TITLE
add test with logo

### PR DESCRIPTION
**Pull request by Tzach123**

### CODE REVIEW

File: src/services/beer-api.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 export const GetBeerApi = async () => {
-  const res = await fetch('MyAPI.json',
-    {
-      headers: {
-        'Content-Type': 'application/json',
-        'Accept': 'application/json'
-      },
-    }
-  );
+  const res = await fetch("MyAPI.json", {
```
Comment (ariel22411) [inline src/services/beer-api.js:2]: What is

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```
Comment (ariel22411) [inline src/App.test.js:6]: Please so a good test

**Review by ariel22411 — CHANGES_REQUESTED** [review]

File: src/services/beer-api.js (hunk 1 above)
Comment (Tzach123) [reply src/services/beer-api.js:2]: done

File: src/App.test.js (hunk 2 above)
Comment (Tzach123) [reply src/App.test.js:6]: done

**Review by ariel22411 — APPROVED** [review]
Configuration

**Pull request merged**